File: Arrays/SingleDimension.c
#include <stdio.h>

int main() {
    int n[10], i;

    printf("Enter 10 numbers (between 1 to 10):\n");
    for(i = 0; i < 10; i++) {
        scanf("%d", &n[i]);
    }

    printf("You entered: ");
    for(i = 0; i < 10; i++) {
        printf("%d ", n[i]);
    }

    return 0;
}

File: Arrays/SingleDimension.js

let number = [];

for(let i = 0; i< 10 ; i++ ) {
    let input = prompt(`Enter the number between 0 and 10 ${i+1}`);
    number[i] = Number(input); 
}
console.log("You Entered This Numbers")

for(let i = 0; i< number.length ; i++ ) {
    console.log(`numbers [${i}] = ${number[i]}`);
}


// Let's try it again tommorow; 